mediator/med_phases_post_ice_mod.F90

```diff
@@ -59,18 +59,6 @@ subroutine med_phases_post_ice(gcomp, rc)
     call med_fraction_set(gcomp, rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-    ! map ice to atm - scaling by updated ice fraction
```

Comment:
I am not sure why this is removed. Is it related with CICE albedos and their usage in FV3?

Reply:
No, this is because there is already a mapping of ice to atm in the post_atm phase. So this was a duplicate mapping.